.env.example/backend.env

```diff
@@ -6,9 +6,6 @@ PORT=3001
 HOST=0.0.0.0
 FRONTEND_URL=http://localhost:3000
 #
-# AUTHENTICATION
-SECRET_KEY={ENTER YOUR SECRET KEY HERE}
```

Comment:
you delete a lot of think
left them. you should bring/change only change that you need in your task